modules/virt-preventing-workload-updates-during-eus-update.adoc

```diff
@@ -0,0 +1,164 @@
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "True",
+    "type": "ReconcileComplete"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T20:30:10Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "True",
+    "type": "Available"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T20:30:10Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "False",
+    "type": "Progressing"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T16:39:11Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "False",
+    "type": "Degraded"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T20:30:10Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "True",
+    "type": "Upgradeable"
+  }
+]
+----
+====
+
+. Manually update your cluster from the source EUS version to the next minor version of {product-title}:
++
+[source,terminal]
++
+====
+$ oc adm upgrade
+====
++
+.Verification
+* Check the current version by running the following command:
++
+[source,terminal]
+----
+$ oc get clusterversion
+----
++
+[NOTE]
+====
+Updating {product-title} to the next version is a prerequisite for updating {VirtProductName}. For more details, refer to the "Updating clusters" section of the {product-title} documentation.
+====
+
+. If you use the recommended *Automatic* approval strategy, {VirtProductName} automatically updates to the corresponding version after you update {product-title}. Monitor the {VirtProductName} update by running the following command:
++
+[source,terminal]
+----
+$ oc get csv -n openshift-cnv
+----
++
+[NOTE]
+====
+If you used the *Manual* approval strategy, you must approve the update in the web console.
+====
+
+. After you confirm that {VirtProductName} successfully updated to the non-EUS version, wait until the `HyperConverged` Operator has the `Upgradeable` status before you perform the next update. Enter the following command and monitor the output:
++
+[source,terminal]
+----
+$ kubectl get hco kubevirt-hyperconverged -n openshift-cnv -o json | jq ".status.conditions"
+----
+
+. Update {product-title} and {VirtProductName} to the target EUS version, monitoring as in the preceding steps to ensure that the updates succeed.
+
+. Restore the workload update methods configuration that you backed up:
++
+[source,terminal]
+----
+$ oc patch hco kubevirt-hyperconverged -n openshift-cnv --type json -p "[{\"op\":\"add\",\"path\":\"/spec/workloadUpdateStrategy/workloadUpdateMethods\", \"value\":$WORKLOAD_UPDATE_METHODS}]"
+----
++
+.Example output
+[source,terminal]
+----
+hyperconverged.hco.kubevirt.io/kubevirt-hyperconverged patched
+----
++
+.Verification
```

Comment:
Should this be a `.Verification` in the middle of a procedure? Looks a bit odd to me.